Symfony/CS/Fixer/Contrib/AlignDoubleArrowFixer.php

```diff
@@ -139,7 +139,7 @@ private function injectAlignmentPlaceholders(Tokens $tokens, $startAt, $endAt)
                 continue;
             }
 
-            if ($token->equals(',')) {
+            if ($token->equals(',') && !in_array($tokens[$index - 1]->getContent(), array(')', ']'), true)) {
```

Comment:
@keradus @GrahamCampbell tests added. Hope this is what you want. Tell me what to do if not. :+1: